src/jit/codegenarmarch.cpp

```diff
@@ -2791,7 +2791,8 @@ void CodeGen::genJmpMethod(GenTree* jmp)
                 GetEmitter()->emitIns_R_S(ins_Load(loadType), loadSize, argReg, varNum, 0);
 
                 // Update argReg life and GC Info to indicate varDsc stack slot is dead and argReg is going live.
-                // Note that we cannot modify varDsc->lvRegNum here because another basic block may not be expecting it.
+                // Note that we cannot modify varDsc->GetRegNum() here because another basic block may not be expecting
+                // it.
```

Comment:
could you please fix this line as well? When they are formatted like this it looks like they are separate block comments.

Reply:
done

Reply:
Thank you.